### ISSUE TITLE
kubernetes-cli hash mismatch

### ISSUE THREAD

**Issue by daenney** [Homebrew/homebrew-core#8132]
```
[I] 14:53:08 ~ $ brew upgrade
==> Upgrading 1 outdated package, with result:
kubernetes-cli 1.5.1_1
==> Upgrading kubernetes-cli
==> Downloading https://homebrew.bintray.com/bottles/kubernetes-cli-1.5.1_1.sierra.bottle.tar.gz
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 265f53a092c76c7c152cf342717acd321edec025671d42a4e6b7b5cae041d945
Actual: 69ec4b4aa3f207591a7d149cb271ffdc318009e9d4eb1d139dfdc7261ba69cad
Archive: /Users/daenney/Library/Caches/Homebrew/kubernetes-cli-1.5.1_1.sierra.bottle.tar.gz
To retry an incomplete download, remove the file above.
```

This has started happening with 1.5.1_1 since #8115 was resolved.

**Comment by nijikon** [comment]
Already on it ;)

**Comment by nijikon** [comment]
Fixed in 7b1a7ff784860503e42cc98057c0ae2933384b7f
